openshift/templates/images/api/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:18.14.0-alpine3.17
+FROM node:lts-alpine
```

Comment:
Can't remember how these Dockerfiles are used. Is it just locally? Just wondering if there's a reason why we specified Alpine 3.17 before.
Also it would be nice to automatically use the latest lts version but if that involves a major version bump it might cause breaking changes.

Reply:
I checked the tags on Docker hub - lts-alpine is v18.18.0 of node, using the Alpine variant of linux (small file size). the alpine version (of the linux vm for the node container itself) for this tag is 3.18

I don't think it's significant whether we use 3.17 or 3.18, based on what I read, but maybe @ShawnTurple can weigh in on this